api/src/main/java/org/eclipse/microprofile/openapi/models/security/SecurityRequirement.java

```diff
@@ -98,10 +98,10 @@ default boolean hasScheme(String securitySchemeName) {
     }
 
     /**
-     * Returns a path item for a given name. This is a convenience method for <code>getSchemes().get(name)</code>
+     * Returns a list of scopes for a given scheme name. This is a convenience method for <code>getSchemes().get(name)</code>
      * 
      * @param securitySchemeName the name of security scheme
-     * @return the corresponding path item or null.
+     * @return a list of scopes or null.
      */
     default List<String> getScheme(String securitySchemeName) {
```

Comment:
shouldn't this method be called `getScopes`?

Reply:
Well `getScheme()` in `SecurityRequirement` is consistent with all the other methods: `addScheme(..)`, `getSchemes()`, ...
I think it is fine the way it is.

Reply:
I agree with @jmini

Reply:
It's true that it's consistent - and this is 100% fine for the purposes of this PR.  

We may want to take a look for `2.0` on the usage of these methods and if they make sense.  For example, the `schemes` here are simple references to `Scheme` objects defined elsewhere, with the addition of which scopes (if oauth based) apply.  So in `getScheme` we're not getting back the original `Scheme` object, we're just getting the `scopes` (if any) we applied to it.